.README/INSTALL.md

```diff
@@ -55,7 +55,7 @@ point in the future so users won't have to do this manual process.
 
 After installing the addon using the steps above, find the `res://addons/IsometricMap/GdnsScripts/` directory and open each
 `.gdns` file located within. This should bring up the file properties in the inspector. For each file, proceed to check,
-and then uncheck, the `Singleton` property.
+and then uncheck, the `Singleton` property:  
```

Comment:
I forgot to re-add newline spaces when removing the text that was originally below this and there were some formatting issues with the text and image falling in-line. Wanted to make sure this was resolved.